content/en/docs/reference/kube-scheduler/policies.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Kube-scheduler Policies
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+A scheduling Policy can be used to specify the predicates and priorities that
+[kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/)
+runs to [filter and score nodes](/docs/concept/scheduling/kube-scheduler/#kube-scheduler-implementation),
+respectively.
+
+A policy can be specified through a file (via `--policy-config-file` flag) or a
+ConfigMap (via `--policy-configmap` flag) that follows the
```

Comment:
nit: The Kubernetes docs style guide discourages using `via`.
Possibly adapt text to something like:

Set your [or You can set a ] scheduling policy by running `kube-scheduler --policy-config-file <filename>` or `kube-scheduler --policy-configmap <ConfigMap>` .

How does the Policy type fit in with the different flags? The link to Policy type returns a 404 (may/may not be fixed with the release. Is there stable documentation link for this information?).

Reply:
Policy takes precedence over most flags.

There is no stable documentation for the kube-scheduler config types.